jans-auth-server/server/src/main/java/io/jans/as/server/model/authorize/JwtAuthorizationRequest.java

```diff
@@ -178,9 +182,6 @@ public JwtAuthorizationRequest(AppConfiguration appConfiguration, AbstractCrypto
                 if (sigAlg == null) {
                     throw new InvalidJwtException("The JWT algorithm is not supported");
                 }
-                if (!validateSignature(cryptoProvider, sigAlg, client, signingInput, encodedSignature)) {
```

Comment:
We can't remove built-in signature validation for jwt authorization request and left it unchanged in other places. What is the goal of removing validation ?